azure-pipelines.yml

```diff
@@ -48,12 +48,7 @@ stages:
       jobs:
       - job: Windows
         pool:
-          ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-            name: NetCore1ESPool-Public
-            demands: ImageOverride -equals Build.Windows.10.Amd64.VS2017.Open
-          ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-            name: NetCore1ESPool-Internal
-            demands: ImageOverride -equals Build.Windows.10.Amd64.VS2017
+          vmImage: 'windows-latest'
```

Comment:
I see that the sourcelink official build is doing signing, which won't work in the hosted pools. You should probably keep the ifs, and only use hosted for the public CI and PR builds.

Reply:
I see.